packages/react-devtools-extensions/src/main.js

```diff
@@ -107,6 +108,10 @@ function createPanelIfReactLoaded() {
 
       const tabId = chrome.devtools.inspectedWindow.tabId;
 
+      registerEventLogger((_event: LogEvent) => {
```

Comment:
I wanted to add that I quite like this pattern you've come up with. It's nice to inject via an import like this vs the way we inject other things (prop passed to `DevTools` which gets shared via context).

I wonder if there are other things we could refactor to be more like this? :)

Reply:
i'll update the RN standalone build. the reason i added an underscore is to avoid the lint rule of unused variables, but I think it might actually not apply here